tasks/install-redhat.yml

```diff
@@ -20,7 +20,7 @@
 
 - name: install driver packages
   yum:
-    name: "{{ nvidia_driver_package_version | ternary('nvidia-driver-latest-dkms-'+nvidia_driver_package_version, 'nvidia-driver-latest-dkms') }}"
+    name: "{{ nvidia_driver_package_version | ternary('nvidia-driver-branch-'+nvidia_driver_package_version, 'nvidia-driver-branch-450') }}"
```

Comment:
You're defaulting to driver branch 450 here directly in the playbook. The behavior we want to have our default be the latest available version of the driver; or if we can't do that, the version should be specified in a variable.